src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -1601,12 +1709,1043 @@ namespace Google.Example.Library.V1.Samples
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Generated code. DO NOT EDIT!
+
+// This is a generated sample ("Request", "test_on_success_map")
+
+// sample-metadata
+//   title: test maps in response handling
+//   description: When there is a very long description for a sample file we can split
+//     the description into multiple lines and the samples will be just fine
+//   usage: dotnet run
+
+using CommandLine;
+
+// [START sample]
+namespace Google.Example.Library.V1.Samples
+{
+    using Google.Example.Library.V1;
+    using System;
+
+    public class GetBookRequestTestOnSuccessMap
+    {
+        /// <summary>
+        /// When there is a very long description for a sample file we can split
+        /// the description into multiple lines and the samples will be just fine
+        /// </summary>
+        public static void SampleGetBook()
+        {
+            LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
+            GetBookRequest request = new GetBookRequest
+            {
+                BookName = new BookName("[SHELF_ID]", "[BOOK_ID]"),
+            };
+            Book response = libraryServiceClient.GetBook(request);
+            string intKeyVal = response.MapStringValue[123];
+            string booleanKeyVal = response.MapBoolKey[true];
+            int mapValueField = response.MapMessageValue["key"].Field;
+            Console.WriteLine($"Test printing map fields: {response.MapListValueValue["quoted_key"]}");
+            Console.WriteLine($"Test printing enum fields of a map value: {response.MapMessageValue["key"].Alignment}");
+        }
+        // [END sample]
+        public static void Main(string[] args)
+        {
+            SampleGetBook();
+        }
+    }
+}
+============== file: Google.Example.Library.V1/Google.Example.Library.V1.Samples/Lovelace.cs ==============
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Generated code. DO NOT EDIT!
+
+// This is a generated sample ("RequestStreamingServer", "prog")
+
+// sample-metadata
+//   title: Programming Books
+//   description: Testing calling forms
+//   usage: dotnet run
+
+using CommandLine;
+
+// [START lovelace]
```

Comment:
The start and end markers are at different levels of code structure.
Does the resultant sample correct this? In this case that would mean adding sufficient `}`s to balance all opening `{`s.

Reply:
Yes! Thanks for the catch! If you don't mind I'll fix this in a separate PR.


Issue filed: https://github.com/googleapis/gapic-generator/issues/2873